make/configure.cmake.h

```diff
@@ -264,6 +265,10 @@
 # ifdef HAVE_FVISIBILITY
 #  define USE_VISIBILITY_ATTRIBUTES
 # endif
+
+# ifdef HAVE_FNOSANITIZE_NULL
+#  define USE_FNOSANITIZE_NULL
```

Comment:
We could remove this define and just use the HAVE_ one above: seems simpler.  I'm guessing the USE_VISIBILITY_ATTRIBUTES was already there with that name historically and the conversion to cmake ended up with the extra define; I don't think we should mirror that for something new.

Reply:
I thought the purpose of creating a new symbol was to limit the usages of `HAVE_FNOSANITIZE_NULL` (which is shared with cmake) to this `configure.cmake.h` file.

Reply:
I see that we don't do the same for other symbols in `configure.cmake.h`. Okay, makes sense to not create a new one here.

Reply:
Done.